kings-cafe/src/routes/privacy/+page.svelte

```diff
@@ -0,0 +1,103 @@
+<main class="container mx-auto py-24 text-lg">
+	<div class="container flex flex-col gap-6 px-4 xl:px-80">
+		<h1 class="text-4xl font-bold uppercase md:hidden">Datenschutz- <br />erklärung</h1>
+		<h1 class="break-words text-5xl font-bold uppercase max-md:hidden">Datenschutzerklärung</h1>
+		<h2 class="text-3xl font-bold uppercase">Wer ist verantwortlich?</h2>
+		<p>
+			Verantwortliche Stelle für diese Seite und damit für die Erhebung deiner Daten ist die
+			Studentenmission in Deutschland (SMD) e.V.:
+		</p>
+		<ul class="border-l-primary border-l-2 px-4 py-2 text-xl">
+			<li>SMD e.V.</li>
+			<li>Universitätsstr. 30</li>
+			<li>35037 Marburg</li>
+		</ul>
+
+		<p>Unseren Datenschutzbeauftragten erreichts du unter:</p>
```

Comment:
```suggestion
		<p>Unseren Datenschutzbeauftragten erreichst du unter:</p>
```
this one just catched my eye